tests/large/test_ide.py

```diff
@@ -31,26 +31,25 @@
 logger = logging.getLogger(__name__)
 
 
-class EclipseIDETests(LargeFrameworkTests):
-    """The Eclipse distribution from the IDE collection."""
-
+class BaseLargeIdeTest(LargeFrameworkTests):
     TIMEOUT_INSTALL_PROGRESS = 120
     TIMEOUT_START = 60
     TIMEOUT_STOP = 60
 
     def setUp(self):
         super().setUp()
-        self.installed_path = os.path.expanduser("~/tools/ide/eclipse")
-        self.desktop_filename = "eclipse.desktop"
+        self.name = ""
+        self.command_name = ""
+        self.installed_path = ""
+        self.desktop_filename = ""
```

Comment:
I personally dislike the need for `self.command_name`. I couldn't dig the source code where the Installer's name, ex. Eclipse EE, gets converted to eclipse-ee. 

Also the fact that we have to manually give strings that at some point will differ than the actual implementation and give   misleading truthy asserts, is a good reason for me to ask - why aren't we instantiating installers such as Eclipse and directly using their properties? The later is the optimal TDD approach but I think there might be a performance reason why we are not doing it?